scripts/lan_diag.ps1

```
@@ -0,0 +1,71 @@
+# Open-FDD LAN connectivity check — run on your Windows PC.
+# Usage:  powershell -ExecutionPolicy Bypass -File .\lan_diag.ps1
+#         powershell -ExecutionPolicy Bypass -File .\lan_diag.ps1 -Host 192.168.204.18 -Port 8765
+
+param(
+    [string]$Host = "192.168.204.18",
```

PSScriptAnalyzer (1.25.0)

[error] The Variable 'Host' cannot be assigned since it is a readonly automatic variable that is built into PowerShell, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)